plugins/flytekit-greatexpectations/flytekitplugins/great_expectations/schema.py

```diff
@@ -196,7 +196,7 @@ def to_python_value(
         ge_conf = GreatExpectationsFlyteConfig(**conf_dict)
 
         # fetch the data context
-        context = ge.data_context.DataContext(ge_conf.context_root_dir)  # type: ignore
+        context = ge.get_context(context_root_dir=self._context_root_dir)  # type: ignore
```

Comment:
should we require `greatexpectations>=1.0.0` in [setup.py](https://github.com/flyteorg/flytekit/blob/master/plugins/flytekit-greatexpectations/setup.py#L9)?

Reply:
I made it 0.18.0 since `get_context` is available in 0.18.X: https://github.com/great-expectations/great_expectations/blob/0.18.0/great_expectations/__init__.py